matching-notification-api/src/main/java/org/phenotips/matchingnotification/match/PatientInMatch.java

```diff
@@ -141,4 +141,10 @@
      * @return What type of access does the user have to this patient profile.
      */
     AccessLevel getAccess();
+
+    /**
+     * @return status of genes (solved or candidate) stored in PatientGenotype class,
+     *         or null if patient does not have genotype.
```

Comment:
I think we should add "...or genotype status is unknown", since this is what we do for genes in MME patients